src/cf-tables.less

```diff
@@ -0,0 +1,484 @@
+                      English, Spanish
+                    </td>
+                    <td data-label="Distance">
+                      1.2 mi
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Eta
+                    </td>
+                    <td data-label="Languages">
+                        English
+                    </td>
+                    <td data-label="Distance">
+                        11.1 mi
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      notes:
+        - Uses '.table__sortable' class to invoke sorting functions
+        - Columns which should be sortable have '.sortable' class applied to the THEAD TH
+        - "Note: The class '.sorted_up' refers to a sort from smallest to greatest
+          (first to last), and '.sorted_down' refers to a sort from greatest to smallest
+          (last to first)."
+        - "Please note the importance of defining a THEAD and TBODY to preserve
+          the table's header through sorting operations"
+        - "Any TH with the '.sortable' class can also be given the 'data-sort_type'
+          attribute"
+        - "In the table example above, Distance is sorted by Number value. See below for
+          the 'data-sort_type' modifier."
+        - "In the table example above, Distance is sorted low-to-high on page load by using
+          the 'sortable__start-up' class in the TH. See the 'sortable__start-up' and
+          'sortable__start-down' modifiers below."
+    - name: data-sort_type (modifier)
+      codenotes:
+        - |
+          <table class="table__sortable">
+            ...
+              <th class="sortable" data-sort_type="number">Column Name</th>
+            ...
+          </table>
+      notes:
+        - "By setting the 'data-sort_type' attribute to 'number', the column is sorted by
+          Number value"
+        - "By default, columns will be sorted as strings."
+    - name: .sortable__start-up, .sortable__start-down (modifiers)
+      codenotes:
+        - |
+          <table class="table__sortable">
+            ...
+              <th class="sortable sortable__start-up">Column Name</th>
+            ...
+          </table>
+      notes:
+        - "To sort the table on page load, use the '.sortable__start-up' and '.sortable__start-down'
+          classes."
+  tags:
+    - cf-tables
+*/
+
+// Color variables
+@table__head-bg:              #f8f8f8;
+@table__cell-bg:              #ffffff;
+@table__cell-bg_alt:          #f8f8f8;
+@table__scrolling-border:     #e3e4e5;
+@table__border:               #babbbd;
+
+// .table
+
+table {
+    .table__right-align {
+        text-align: right;
+    }
+}
+
+.table-wrapper__scrolling {
+    box-sizing: border-box;
+    overflow-x: auto;
+    overflow-y: hidden;
+    width: 100%;
+    table {
+        border: 1px solid @table__scrolling-border;
+        .table__striped();
```

Comment:
Unforeseen consequence - removing the `.table__striped` CSS here and making it a class for a `.table-wrapper__scrolling` table means that at small widths, the stripes disappear (as is standard for the `.table__striped` class, because stacked tables aren't supposed to be striped.)

So, I can add overrides to the small screen section, or I can leave the `.table__striped()` in here. Thoughts?